### PR TITLE
comment error

### PR DESCRIPTION
**Pull request by StudyNick**
spelling mistakes, "syou" is error, "you" is right

### CODE REVIEW

**Comment by StudyNick** [comment]
I signed it!

**Comment by k82cn** [comment]
@k8s-bot ok to test

File: pkg/controller/client_builder.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ import (
 	"github.com/golang/glog"
 )
 
-// ControllerClientBuilder allow syou to get clients and configs for controllers
+// ControllerClientBuilder allow you to get clients and configs for controllers
```
Comment (mikedanese) [inline pkg/controller/client_builder.go]: Should be 'allows you'

Comment (StudyNick) [inline pkg/controller/client_builder.go]: yes, 	update file

**Comment by mikedanese** [comment]
@StudyNick please squash

**Comment by StudyNick** [comment]
@mikedanese finish

**Comment by mikedanese** [comment]
This is still two commits.

**Comment by k82cn** [comment]
@StudyNick , would you merge into one commit?

**Pull request closed without merging**